.github/workflows/ci-weekly.yml

```diff
@@ -162,4 +195,6 @@ jobs:
           ${{github.workspace}}/packages/utilities/fast-web-utilities/coverage/lcov.info:lcov
           ${{github.workspace}}/packages/utilities/fast-eslint-rules/coverage/lcov.info:lcov
           ${{github.workspace}}/sites/fast-component-explorer/coverage/lcov.info:lcov
-        debug: false
\ No newline at end of file
+        debug: false
+    
+    
```

Comment:
```suggestion
        debug: false

```

Reply:
Not sure why I can't commit this now... Will address